presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -67,7 +67,8 @@ statement
     | SHOW SCHEMAS ((FROM | IN) identifier)? (LIKE pattern=STRING)?    #showSchemas
     | SHOW CATALOGS (LIKE pattern=STRING)?                             #showCatalogs
     | SHOW COLUMNS (FROM | IN) qualifiedName                           #showColumns
-    | SHOW STATS (FROM | IN)? qualifiedName                            #showStats
+    | SHOW STATS (FOR | ON)? qualifiedName                             #showStats
+    | SHOW STATS (FOR | ON)? '(' querySpecification ')'                #showStatsParametrized
```

Comment:
It was to mean that filtering "parameter" as query specification are allowed. I agree name could be better.
I will change it to "showStatsForQuery"